src/Bundle/test/app/AppKernel.php

```diff
@@ -22,29 +22,39 @@ class AppKernel extends Kernel
      */
     public function registerBundles()
     {
-        return [
+        if ('prod' === $this->getEnvironment()) {
+            $loader = require __DIR__ . '/../../../../vendor/autoload.php';
+            $loader->addPsr4('AppBundle\\', __DIR__ . '/../src/AppBundle/');
+        }
+
+        $bundles = [
             new Symfony\Bundle\FrameworkBundle\FrameworkBundle(),
-            new FOS\RestBundle\FOSRestBundle(),
-            new JMS\SerializerBundle\JMSSerializerBundle(),
             new Doctrine\Bundle\DoctrineBundle\DoctrineBundle(),
             new Sylius\Bundle\ResourceBundle\SyliusResourceBundle(),
             new BabDev\PagerfantaBundle\BabDevPagerfantaBundle(),
-            new Bazinga\Bundle\HateoasBundle\BazingaHateoasBundle(),
             new Symfony\Bundle\TwigBundle\TwigBundle(),
-            new \winzou\Bundle\StateMachineBundle\winzouStateMachineBundle(),
+            new winzou\Bundle\StateMachineBundle\winzouStateMachineBundle(),
             new Stof\DoctrineExtensionsBundle\StofDoctrineExtensionsBundle(),
-            new Fidry\AliceDataFixtures\Bridge\Symfony\FidryAliceDataFixturesBundle(),
-            new Nelmio\Alice\Bridge\Symfony\NelmioAliceBundle(),
             new AppBundle\AppBundle(),
         ];
+
+        if (in_array($this->getEnvironment(), ['dev', 'test'])) {
```

Comment:
Why don't we always add those bundles? They're always included when installed with `require-dev` dependencies.

Reply:
```suggestion
        if (in_array($this->getEnvironment(), ['dev', 'test'], true)) {
```